crowbar_framework/app/views/barclamp/neutron/_edit_attributes.html.haml

```diff
@@ -14,8 +14,11 @@
 
       = select_field :networking_plugin, :collection => :networking_plugins_for_neutron
 
-      #mode_container
-        = select_field :networking_mode, :collection => :networking_modes_for_neutron
+      #ml2_mechanism_drivers_container
+        = string_field %w(ml2_mechanism_drivers)
```

Comment:
Should this really be a string field? Could we have something more magic where people can check the bits they want?